src/components/menu/menu.scss

```diff
@@ -21,16 +24,29 @@
   }
 }
 
+body[data-theme="dark"] .actmenu {
+  --bg1: rgba(44, 44, 44, 0.8);
+  --bg2: rgba(255, 255, 255, 0.08);
+
+  .menuopt img {
+    filter: hue-rotate(180deg) invert(1);
```

Comment:
This is useless

Reply:
Useles will be a harsh word, I would say it is replacable to something better.